build.gradle

```diff
@@ -254,13 +258,19 @@ afterEvaluate { project ->
     // debugging
     task printCompilerInput << {
       logger.info "**JAVAC INPUT FILES**"
-      compileReleaseJava.source.each {
+        compileReleaseJavaWithJavac.source.each {
         logger.info '  ' + it.absolutePath
       }
       logger.info "**JAVAC CLASSPATH**"
-      compileReleaseJava.classpath.each {
+        compileReleaseJavaWithJavac.classpath.each {
         logger.info '  ' + it.absolutePath
       }
     }
-    compileReleaseJava.dependsOn printCompilerInput
+    compileReleaseJavaWithJavac.dependsOn printCompilerInput
 }
+
+allprojects {
+    repositories {
+        mavenCentral()
+    }
```

Comment:
This seem to duplicate L27-L33, please double check and try to keep the list of repositories used to fetch project dependencies in one place only.

Note there is another configuration for build dependencies on L4-L6. I don't know if it's possible to unify them with project dependencies, let's leave that out of scope of current changes.